crates/sqlexec/src/planner/session_planner.rs

```diff
@@ -1032,22 +1034,57 @@ impl<'a> SessionPlanner<'a> {
 
         let mut m = stmt.options;
 
-        validate_ident(&stmt.dest)?;
         let dest = normalize_ident(stmt.dest);
 
+        let (dest, uri) = if matches!(
+            dest.as_str(),
+            CopyToDestinationOptions::LOCAL
+                | CopyToDestinationOptions::GCS
+                | CopyToDestinationOptions::S3_STORAGE
+        ) {
```

Comment:
I need to find a cleaner way to do this. I mean, this `match` statement and the one below (the main one which matches with the destinations) both use strings. Things may be left out when adding more Copy-To destinations. Not the worst thing in this world, though, so I'm happy to let it go through.

Reply:
🤷

This looks readable enough for now.